examples/verrazzano/scripts/install_verrazzano_operator.template.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2017, 2021 Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+echo "Installing Verrazzano Enterprise Container Platform"
+
+kubectl apply -f https://github.com/verrazzano/verrazzano/releases/download/v${verrazzano_version}/operator.yaml
```

Comment:
Missing double quotes around variable

Reply:
This will be rendered by Terraform. The quotes are not needed here, otherwise it will be treated as a string.

Reply:
Ah, ok.